packages/renderer/src/lib/preferences/PreferencesConnectionCreationOrEditRendering.svelte

```diff
@@ -184,11 +184,7 @@ onMount(async () => {
     }
   }
   if (!connectionInfo) {
-    try {
```

Comment:
I wasn't sure if I should also remove the try catch block in line 215 since it is a bit different. Is there a special case there that I might be missing?

Reply:
No you should keep the try / catch as the optional audit method from a provider may throw an exception so we should be protected